Orange/canvas/canvas/view.py

```diff
@@ -63,6 +66,32 @@ def mouseReleaseEvent(self, event):
 
         return QGraphicsView.mouseReleaseEvent(self, event)
 
+    def reset_zoom(self):
+        self.__set_zoom(10)
+
+    def change_zoom(self, delta):
+        self.__set_zoom(self.__scale + delta)
+
+    def __set_zoom(self, scale):
+        self.__scale = min(15, max(scale, 3))
+        transform = QTransform()
+        transform.scale(self.__scale / 10, self.__scale / 10)
+        self.setTransform(transform)
+
+    def wheelEvent(self, event: QWheelEvent):
+        # use mouse position as anchor while zooming
+        self.setTransformationAnchor(2)
+        if event.modifiers() & Qt.ControlModifier:
+            delta = event.angleDelta().y()
+            if QT_VERSION >= 0x050300 \
+                    and event.source() != Qt.MouseEventNotSynthesized \
```

Comment:
[http://doc.qt.io/qt-5/qwheelevent.html#source](http://doc.qt.io/qt-5/qwheelevent.html#source) says it was introduced in Qt 5.5.

Reply:
I changed this to 0x050500. The [documentation for the corresponding enum](http://doc.qt.io/qt-5/qt.html#MouseEventSource-enum) says that the enum exists from 5.3. I don't know how would you get the source then.